src/core/symbology-ng/qgsrendererv2.h

```diff
@@ -61,6 +61,18 @@ class CORE_EXPORT QgsSymbolV2LevelItem
     int mLayer;
 };
 
+class CORE_EXPORT QgsRenderOptions
```

Comment:
Will do as soon as we know what the final name will be - and if it even stays :)